### PR TITLE
Version 1.9.5

### PR DESCRIPTION
**Pull request by tevincent**
What's new in 1.9.5
* Upgrade build tools to 8.1.3
* Upgrade gradle to 8.0
* Delete package from AndroidManifest.xml and add namespace to build.gradle
* Upgrade androidx.collection to 1.3.0

### CODE REVIEW

**Review by sirambd — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #Sat May 21 18:52:48 EDT 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-all.zip
```
Comment (sirambd) [inline gradle/wrapper/gradle-wrapper.properties]: try with 8.4

**Review by sirambd — CHANGES_REQUESTED** [review]
Waiting for latest gradle version

**Review by sirambd — APPROVED** [review]

**Comment by odmfl** [comment]
@tevincent @sirambd
Hi, can you implement text search in PDF in pdfium and pdfview?  This open source app based on pdfium and android pdf viewer allows you to do this, it would be great if you implemented it in your libraries

https://gitlab.com/mudlej_android/mj_pdf_reader

https://gitlab.com/mudlej_android/mj_pdf_reader/-/commit/a0d58ab8f72ea8bfe4c85cec52069c7a3e861be3

**Comment by VincentTe** [comment]
Hi @odmfl , yes this is on my to-do list so it'll come out soon 🙂.

**Pull request merged**